lib/mail/utilities.rb

```diff
@@ -221,5 +221,24 @@ def map_with_index( enum, &block )
 
     end
 
+    # Returns true if the object is considered blank.
+    # A blank includes things like '', '   ', nil,
+    # and arrays and hashes that have nothing in them.
+    # 
+    # This logic is mostly shared with ActiveSupport's blank?
+    def self.blank?(string)
+      if string.is_a?(NilClass)
+        true
+      elsif string.is_a?(String)
```

Comment:
@jeremy I switched to ```kind_of?```, but I don't think it mattered... I think ```is_a?``` is a synonym.

http://stackoverflow.com/questions/3893278/ruby-kind-of-vs-instance-of-vs-is-a